students/elmar_m/lesson07/html_render.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+'''
+file: html_render.py
+elmar_m / 22e88@mailbox.org
+Lesson07: HTML renderer
+'''
+
+class Element:
+    tag = 'element'
+    props = []
+    propstring = ''
```

Comment:
I don't think that `props` and `propstring` need to be initialized as class attributes with empty values since you're only using them to build the `tagopen` attribute